business_objects/record_label_association.py

```diff
@@ -92,6 +92,24 @@ def get_tokens(project_id: str) -> List[Any]:
     return general.execute_all(query)
 
 
+def get_manual_tokens_by_record_id(
+    record_id: str,
```

Comment:
add project_id to arguments and use it to filter to improve performance

Reply:
done